packages/td-tools/test/ThingModelHelperTest.ts

```diff
@@ -489,7 +486,7 @@ class ThingModelHelperTest {
 
         await expect(this.thingModelHelpers.getPartialTDs(thing)).to.be.rejectedWith(
             Error,
-            "https status code not 200 but 404 for https://example.com/models/colored-lamp-1.0.0.tm.jsonld"
+            "https status code not 200"
```

Comment:
This PR includes now changes that belong to another PR

Reply:
Sorry about that. I forgot to pull the master beforehand. Now only this PR changes are included